src/app/app.component.ts

```diff
@@ -39,9 +39,8 @@ export class AppComponent implements OnInit {
         this.isActive(event.url);
       }
     });
-    this.ping.setDefaultPingTime();
-    const pingInterval = JSON.parse(localStorage.getItem('PING_INTERVAL'));
-    this.ping.pingIntervalChanged.next(pingInterval);
+    this.setPingIntervalOnAppLaunch();
+    this.setGraphRefreshIntervalOnAppLaunch();
```

Comment:
which Graph, you mean DashboardGraph or StasHistoryGraph?  there are assetReadingsGraph too in app